src/app/card/card.component.html

```diff
@@ -4,16 +4,16 @@
 
   <div class="card-content {{ textClass }}">
 
-    <span class="card-title">
+    <div class="card-title" *ngIf="hasCardTitle">
```

Comment:
IMO using `*ngIf="title"` is the best option and would to the same here with less code